src/lazyLoad/next_js/next_GetJSScript.ts

```
@@ -0,0 +1,108 @@
+
+    // iterate through script tags
+    for (const scriptTag of scriptTags) {
+        // get the src attribute
+        const src = $(scriptTag).attr("src");
+
+        // see if the src is a JS file
+        if (
+            src !== undefined &&
+            src.match(
+                /(https:\/\/[a-zA-Z0-9_\_\.]+\/.+\.js\??.*|\/.+\.js\??.*)/
+            )
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)